lib/samples/apply_unique_value_renderer/README.metadata.json

```diff
@@ -2,10 +2,21 @@
     "category": "Visualization",
     "description": "Render features in a layer using a distinct symbol for each unique attribute value.",
     "ignore": false,
-    "images": [],
-    "keywords": [],
+    "images": ["apply_unique_value_renderer.png"],
+    "keywords": [
+        "draw", "renderer", "symbol", "symbology", "values"
```

Comment:
keywords should contain the relavant_apis too.

You can actually delete this file and use the script `tools/readme_scripts/metadata_checker.py` to generate a new one based on the README.md

Reply:
actually, it is `create_metadata_from_README.py -s` to create the file

Reply:
oops... my bad.